src/vs/workbench/parts/execution/common/terminal.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file should live in `src/vs/workbench/parts/execution/common/node/terminal.ts` as it's not usable in a browser. Check out [this wiki page](https://github.com/Microsoft/vscode/blob/master/wiki/contributing/code-organization.md#target-environments) for more information on code organization, basically things in common/ and browser/ need to work both in vscode (under node) and in monaco (on a browser).